### DISCUSSION TITLE
Question: why can't the table name for xls begin with digit?

### DISCUSSION THREAD

**Discussion by dongyi-cai-windsab** [ClosedXML/ClosedXML#1990] [Q&A] [answered]
I can rename the table name inside xlsx file with pure digits just fine, why can't I do it in code?

The error message is:

The table name '15' doesn't begin with a letter, an underscore or a backslash

**Comment by jonathanIvorBetts** [comment, marked as answer]
> I can rename the table name inside xlsx file with pure digits just fine, why can't I do it in code?

You sure? The limitation appears to be derived from excel...
![image](https://github.com/ClosedXML/ClosedXML/assets/52923016/54fc199e-d1b4-45b8-9477-ffde70a1f225)
